### ISSUE TITLE
Window auto-resizes during playback

### ISSUE THREAD

**Issue by GitTheHellOutaHere** [lhc70000/iina#1039]
Twice in the past week the following has occured:

I had a video playing in fullscreen and went into the kitchen to do something else (I can still see the screen). While in there the player window resized from fullscreen to its default opening size.

I had to drop what I was doing and return to the computer to enter fullscreen again.

No one else is here (unless I have a poltergeist).

- OS X 10.12.6
- IINA 0.0.12

**Comment by GitTheHellOutaHere** [comment]
As it turns out, I *DO* have a poltergeist.

Well, actually more of a spy, and her name is Siri.

Tonight I was watching an episode of a tv show in fullscreen, and twice either the Audio or Video Effects Settings window (I didn't notice which one before I closed them)  opened on its own.

And then Pages launched on its own.

But it wasn't until the Dictation Commands command window suddenly opened when I realized that Siri had been eavesdropping.

At one point in the show a character says "HEY! You guys are here, the fun continues...nay increases."
And a second or two later the IINA window DECREASED.


I replayed that section several times and about half the time Siri decreased the IINA window size.

Then I put on the headset and repeated the dialog. Twice I was able to activate Siri and resize the window.


Earlier this week I read an article about the many ways Siri is useful and how to enable Siri using a voice command of **HEY** (or any other word you choose).

My 'puter doesn't have a mic, but sometimes I leave my headset plugged in and hanging on the side of the desk. That puts the headset mic about 34" away from my desktop speakers, and obviously within range.

And since Siri tattles to Apple about everything "she" hears, I'm sure glad I was watching a tv show and not...oh, I don't know...PORN!

I see a major security risk here by enabling Siri by voice instead of a keyboard command (or keybinding, as IINA calls them).
The article should be edited to state that users should choose an UNCOMMON activation word instead of "Hey."

For now the impish poltergiest in my home has been exorcised.
